src/graphql-client.ts

```diff
@@ -20,21 +18,9 @@ const headers = {
 
 const uri = "https://api.github.com/graphql";
 
-const fragmentMatcher = new IntrospectionFragmentMatcher({
-    introspectionQueryResultData: {
-        __schema: {
-            types: []
-        }
-    }
-});
-const cache = new InMemoryCache({ fragmentMatcher });
-const link = new HttpLink({
-    uri,
-    headers,
-    fetch
-});

```

Comment:
All apollo-link, apollo-link-http, and apollo-link-http-common functionality is included in the @apollo/client package https://www.apollographql.com/docs/react/migrating/apollo-client-3-migration/#apollo-link-and-apollo-link-http:~:text=All%20apollo%2Dlink%2C%20apollo%2Dlink%2Dhttp%2C%20and%20apollo%2Dlink%2Dhttp%2Dcommon%20functionality,is%20included%20in%20the%20%40apollo%2Fclient%20package